src/haddock/modules/analysis/seletopclusts/__init__.py

```diff
@@ -59,10 +59,15 @@ def _run(self):
                                 and pdb.clt_model_rank == model_ranking
                                 ):
                             self.log(
-                                f"Selecting {pdb.file_name} "
-                                f"Cluster {target_ranking} "
-                                f"Model {model_ranking}"
+                                f" {pdb.file_name} "
```

Comment:
```suggestion
                                f"{pdb.file_name} "
```
Better not to have two spaces here right?